### PR TITLE
test(discover): Skip invalid test

### PR DESCRIPTION
**Pull request by lynnagara**
This test isn't valid because it generates a query that attempts to
look at the group_id column in the Discover table. This table is not
present in the Discover table since it is not relevant for transactions.

It's likely the underlying implementation of has:issue needs to be fixed
though that is not tackled here.

has:issue or issue:unknown seems equivalent to type != transaction since all
non transaction events have issues, and no transaction ever has an issue.

These queries just happened to pass previously because events and transactions
were temporarily colocated in the same table, but that will no longer be the
case going forward.

### CODE REVIEW

**Comment by lynnagara** [comment]
@wmak Picked on you because I know you've seen some of these previously

**Review by dashed — APPROVED** [review]

**Comment by wmak** [comment]
I can look at these, but it looks like Alberto beat me to the punch.
What happens to `has:issue` queries after we switch to the errors table? Do they fail in snuba or do we just get an empty result instead?

**Comment by lynnagara** [comment]
@wmak They don't fail, we get an empty result. But is has:issue really a useful query? Doesn't it just tell us if something is a transaction or not? Which would then be a query we can send to the Discover dataset.

File: tests/snuba/api/endpoints/test_organization_events_v2.py [hunk 1]
```diff
@@ -410,12 +411,12 @@ def test_has_issue(self):
...
-        response = self.do_request(query, features=features)
-        assert response.status_code == 200, response.content
-        assert len(response.data["data"]) == 1
-        assert response.data["data"][0]["issue"] == "unknown"
+        # # should only show 1 event of type transaction since they dont have issues
+        # query = {"field": ["project", "issue"], "query": "!has:issue", "statsPeriod": "14d"}
+        # response = self.do_request(query, features=features)
+        # assert response.status_code == 200, response.content
+        # assert len(response.data["data"]) == 1
+        # assert response.data["data"][0]["issue"] == "unknown"
```
Comment (wmak) [inline tests/snuba/api/endpoints/test_organization_events_v2.py:419]: Let's just delete this instead of commenting it out

**Review by wmak — APPROVED** [review]
oh I don't expect it to be useful, I just know users try strange queries all the time and I wanted to make sure we don't start erroring when they do that.

this LGTM too though :+1:

File: tests/snuba/api/endpoints/test_organization_events_v2.py (hunk 1 above)
Comment (lynnagara) [reply tests/snuba/api/endpoints/test_organization_events_v2.py:419]: Will do, yeah wasn't sure if we wanted to resurrect later.

**Pull request merged**